src/views/Homepage/Acts/index.tsx

```diff
@@ -30,46 +30,58 @@ const Acts: React.FC<Props> = ({ active, setActive }) => {
     }
   }
 
+  /* const Slide = ({ stepIndex, imgSrc }: { stepIndex: number, imgSrc: string }) => (
+    <section
+      className={classNames({
+        [styles.slide]: true,
+        [styles.past]: step > stepIndex,
+        [styles.present]: step === stepIndex,
+        [styles.future]: step < stepIndex,
+      })}
+      // style={{ backgroundImage: `url("${imgSrc}")` }}
+    >
+      <img src={imgSrc} /> // why not moving with section transition?
+    </section>
+  ) */
+
```

Comment:
I tried to make this a component, then found the transition on images would not be showing; neither of `background-image` or nested `img` could work, @robertu7 knows why?

Reply:
It's by re-rendeing, try `React.memo`:

```
type SlideProps = {
  step: number
  stepIndex: number
  imgSrc: string
}

const BaseSlide = ({ step, stepIndex, imgSrc }: SlideProps) => (
  <section
    className={classNames({
      [styles.slide]: true,
      [styles.past]: step > stepIndex,
      [styles.present]: step === stepIndex,
      [styles.future]: step < stepIndex,
    })}
    // style={{ backgroundImage: `url("${imgSrc}")` }}
  >
    <img src={imgSrc} />
  </section>
)

const Slide = React.memo(BaseSlide, (prev: any, next: any) => prev.s === next.s)

const Acts: React.FC<Props> = ({ active, setActive }) => {

...

      <Slide step={step} stepIndex={0} imgSrc="/images/story-1.png" />
      <Slide step={step} stepIndex={1} imgSrc="/images/story-2.png" />
      <Slide step={step} stepIndex={2} imgSrc="/images/story-3.png" />
      <Slide step={step} stepIndex={3} imgSrc="/images/story-4.png" />
```